src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<persistence xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://java.sun.com/xml/ns/persistence
+             http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
+             version="2.0" xmlns="http://java.sun.com/xml/ns/persistence">
+
+    <persistence-unit name="jalaPersistence">
+
+        <!--Aqui se representara las clases-->
+        <class>org.fundacionjala.JalaConverter.model.empleado.Empleado</class>
+        <class>org.fundacionjala.JalaConverter.model.empleado.Direccion</class>
+        <class>org.fundacion.jala.converter.models.User</class>
+        <class>org.fundacion.jala.converter.models.Project</class>
+
+        <properties>
+            <property name="javax.persistence.jdbc.driver" value="com.mysql.cj.jdbc.Driver" />
+            <property name="javax.persistence.jdbc.url" value="jdbc:mysql://127.0.0.1:9090/jalaconverter" />
+            <property name="javax.persistence.jdbc.user" value="root" />
+            <property name="javax.persistence.jdbc.password" value="root" />
+
```

Comment:
Maybe you should erase the blank spaces.

Reply:
ok

Reply:
I agree with you